website/docs/project_settings/settings_project_maya.md

```diff
@@ -0,0 +1,116 @@
+---
+id: settings_project_maya
+title: Project Maya Settings
+sidebar_label: Maya
+---
+
+# Open Workfile Post Initialization
+Open the Workfile on Maya's launch.
```

Comment:
```suggestion
Open the Workfile on Maya's launch. This changes default behaviour of opening last Workfile (that works by passing Workfile as argument when launching Maya). When enabled, last Workfile is loaded after Maya is fully initialized. This can help in certain scenarios where Workfile depends on Maya plugins that are not fully initialized yet when the scene is already opened.
```